cmd/release-notes/generate.go

```diff
@@ -120,7 +120,7 @@ func addGenerateFlags(subcommand *cobra.Command) {
 	subcommand.PersistentFlags().StringVar(
 		&opts.RepoPath,
 		"repo-path",
-		env.Default("REPO_PATH", filepath.Join(os.TempDir(), "k8s-repo")),
+		env.Default("REPO_PATH", filepath.Join(os.TempDir(), fmt.Sprintf("%s-%s", opts.GithubOrg, opts.GithubRepo))),
```

Comment:
This is not going to work at all because `opts.GithubOrg` and `opts.GithubRepo` are set after the `repo-path` flag is defaulted. In other words, this is always going to default to `kubernetes-kubernetes`, so the issue is not solved. We need to take another approach here, one option that comes to my mind (but that might not be ideal) is to leave the default value as empty, and then properly default in the `ValidateAndFinish` function.

Reply:
thanks, forgot to test it with different repo from default